src/Writer/Common.php

```diff
@@ -126,7 +125,7 @@ public function upsert(array $table, $targetTable)
             $this->create($destinationTable);
         }
 
-        $columns = array_map(function($item) {
+        $columns = array_map(function ($item) {
```

Comment:
Chceme to psát takhle? To mi přijde divné.

Reply:
Souhlasím že je to divný, taky to vyždy píšu bez mezery a `phpcs` pak prudí. Tohle je zrovna ale už v PSR-2 https://www.php-fig.org/psr/psr-2/#6-closures

Reply:
Podla mna je to s tou medzerou logickejsie. Aj v definicii "normalnej" funkcie je medzera. :)

Reply:
Právě, že IMHO není. Před závorkou mezera není. Tj. já to čtu jako "mezi definicí a parametry metody není mezera". A v tomhle případě je "definice" = "function". 
```
public function __construct($config, Logger $logger, $configDefinition = null)
---------------------------^
```

Reply:
Ale souhlas, nevěděl jsem, že je to v PSR2 - to nemá cenu rozporovat.

Reply:
mezera je za slovom function, nazov funkcie v pripade anonymnej funkcie chyba :) Takze tam nie je nic.